data/textures/Sfx.ini

```diff
@@ -0,0 +1,6 @@
+damageFile=textures/smoke.png
+smokeFile=textures/Smoke45Frames.dds
+explosionFile=textures/explosions/explosions.dds
+damagePacking=billboard,1,1
+smokePacking=atlas,7,45
+explosionPacking=atlas,6,32
```

Comment:
I'm guessing this file doesn't end in a new line? Maybe it should?
(See, still proofreading!)